server/api/users/users.test.ts

```diff
@@ -24,16 +26,27 @@ describe('auth router', () => {
 
       const res = await server.exec.get(`/api/users/${user.id}`);
       expect(res.status).toBe(200);
-      expect(res.body).toEqual({
-        id: user.id,
-        discord_user_id: user.discord_user_id,
-        discord_name: user.discord_name,
-        bio: user.bio,
-        twitter_username: user.twitter_username,
-        linkedin_url: user.linkedin_url,
-        github_username: user.github_username,
-        website: user.website,
-      });
+      expect(res.body).toEqual(getExpectedUserObject(user));
+      // Delete the user that was created
+      await User.destroy({ where: {}});
+    });
+  });
+
+  describe('GET /', () => {
+    it('returns 404 if specified no users exist', async () => {
```

Comment:
In relation to [tim's previous comment](https://github.com/timmyichen/dev-directory/pull/65/files#r1148667108), this test should probably expect to receive an empty array instead.